openassessment/templates/openassessmentblock/staff_area/oa_staff_assessment.html

```diff
@@ -0,0 +1,52 @@
+{% load tz %}
+{% load i18n %}
+{% spaceless %}
+
+    {% block body %}
+    <div class="ui-toggle-visibility__content">
+        <div class="wrapper--staff-assessment">
+            <div class="step__instruction">
+                <p>{% trans "Allows you to override the current learner's grade using the problem's rubric." %}</p>
+            </div>
+
+            <div class="step__content">
+                <article class="staff-assessment">
+                    <div class="staff-assessment__display">
+                        <header class="staff-assessment__display__header">
+                            <h3 class="staff-assessment__display__title">
+                                {% blocktrans %}
+                                    Response for: {{ student_username }}
+                                {% endblocktrans %}
+                            </h3>
+                        </header>
+
+                        {% include "openassessmentblock/oa_submission_answer.html" with answer=submission.answer answer_text_label="The learner's response to the question above:" %}
```

Comment:
Good question. oa_submission_answer.html does call ``trans`` on the parameter, but I don't know whether the string will be scraped. This convention is used by all the assessments so they will all need to be fixed if not.